bettermdptools/algorithms/rl.py

```diff
@@ -44,6 +45,10 @@ def __init__(self, env):
         #       return np.random.choice(indxs)
         #   else:
         #       return np.random.randint(len(Q[state]))
+        seed = get_seed()
+        if seed is not None:
+            np.random.seed(seed)
```

Comment:
```suggestion
```

This is already done in the seed.py

Reply:
I included it because I noticed the results are different for q learning.
Without calling `np.random.seed(seed)` again: `-0.11308904 mean Q` and `-0.11 mean test_scores`
Calling `np.random.seed(seed)` again: `-0.1253272 mean Q` and `-0.03 mean test_scores`
I thought it would be better to call it again to ensure all the possible randomness are properly seeded but I can remove it if that's not needed.